README.md

```diff
@@ -359,6 +361,42 @@ and the whole idea of property checking is to take that burden
 off the programmer. Despite the theoretical discomfort, this
 approach can turn out to be practical.
 
+### Generating Structs
+
+It is very simple to generate structs in QuickCheck. Consider the following
+example, where the struct `Point` is defined:
+
+```rust
+struct Point {
+    x: i32,
+    y: i32,
+}
+```
+
+In order to generate a random `Point` instance, you need to implement
+the trait `Arbitrary` for the struct `Point`:
+
+```rust
+use quickcheck::{Arbitrary, Gen};
+// Important: Without this import the method gen() of Gen is not available then.
+use rand::Rng;
+
+impl Arbitrary for Point {
+    fn arbitrary<G: Gen>(g: &mut G) -> Point {
+        Point {
+            x: g.gen(),
+            y: g.gen(),
+        }
```

Comment:
This should be
``` rust
Point {
    x: i32::arbitrary(g),
    y: i32::arbitrary(g),
}
```

They way you did it depends on the `rand` api, which is prone to changing, see #241. Making my suggested change also obsoletes the accompanying notice (and the dependency hell that I think would come with it)